modelsets/binary

```diff
@@ -0,0 +1,9 @@
+corellospinmatrix_detector
+erfc_neuron
+ginzburg_neuron
+mcculloch_pitts_neuron
+spin_detector
+spike_generator
+spike_recorder
+static_synapse
+multimeter
```

Comment:
newline at EOF (also in other files in ``modelsets``)

Reply:
Fixed in e5c3b89.